gneiss/plot/tests/test_plot.py

```diff
@@ -87,5 +88,131 @@ def test_visualization_small(self):
                           html)
 
 
+class TestBalanceTaxonomy(unittest.TestCase):
+
+    def setUp(self):
+        self.results = "results"
+        os.mkdir(self.results)
```

Comment:
Nope - because I tear it down right after creation.  The main reason why I do this is because it makes debugging a heck of a lot easier.

It is making the assumption that the current directory is clean and doesn't have the folder `results`.  This is a safe assumption, particularly given that these file directories are created on the fly.